charts/alfresco-audit-storage/Chart.yaml

```
@@ -0,0 +1,20 @@
+apiVersion: v2
+name: alfresco-audit-storage
+description: A Helm chart for Kubernetes to deploy Alfresco Audit Storage
+type: application
+version: 0.0.1-alpha.0
```

```suggestion
version: 0.1.0-alpha.0
```

the first alpha can't be a bugfix release - anyway not a big deal